src/validate/is-module-only-rule.js

```diff
@@ -5,7 +5,8 @@
 function isModuleOnlyRule(pRule) {
   return (
     Object.prototype.hasOwnProperty.call(pRule.from, "orphan") ||
-    Object.prototype.hasOwnProperty.call(pRule.to, "reachable")
+    Object.prototype.hasOwnProperty.call(pRule.to, "reachable") ||
+    Object.prototype.hasOwnProperty.call(pRule.to, "module")
```

Comment:
How come this works? Did you mean ...
```javascript
Object.prototype.hasOwnProperty.call(pRule, "module")
```

Reply:
:-o that fails idd. Not in the last place because it wasn't used at all in the required rule check => fixed

- [ ] slap an extra automated test or two on that

Reply:
fixed in 2351a0b